.gitlab-ci.yml

```diff
@@ -552,6 +555,8 @@ build/amd/nompi/clang/hip/release/static:
     CXX_COMPILER: "clang++"
     BUILD_OMP: "ON"
     BUILD_HIP: "ON"
+    MPI_AS_ROOT: "ON"
+    BUILD_MPI: "ON"
```

Comment:
Yes, OpenMPI developers strictly warn against running mpi as a root user. It seems that if there is a bug or something, then mpirun as root might leave the system itself in a unusable state. But if someone wants to do that nonetheless, then they provide a couple of ways to do so.

For us, it is not an issue because we mainly run inside containers, which is fully isolated, so any corruptions/crashes in the container should not affect the outer system.

Reply:
But you could just run as a regular user, rather than as root, right?

Reply:
No, docker always runs as root.